ui/jstests/empty-test.js

```diff
@@ -0,0 +1,9 @@
+define(['QUnit'], function(QUnit) {
+  'use strict';
+  QUnit.test(
+    'empty test used to prevent build failure due to lack of tests',
+    function(assert) {
+      assert.equal(1, 1);
+    }
+  );
+});
```

Comment:
Your comment was on a module to be tested, not on the test. Also the build will fail if there are zero tests, see the description of this test.